config/schema/acquia_lift.settings.yml

```diff
@@ -98,19 +98,6 @@ acquia_lift.settings:
               type:
                 type: string
                 label: 'Item Type this is mapped to.'
-    thumbnail:
```

Comment:
I think there needs to be some upgrade function that removes this config no? What if it is populated? Will this work?